.github/workflows/axon-alphanet-test.yaml

```
@@ -0,0 +1,22 @@
+name: Migrate Contract to Axon-Alphanet
+
+on:
+  push:
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+
+env:
+  AXON_HTTP_RPC_URL: https://rpc-alphanet-axon.ckbapp.dev/ 
```

```suggestion
  AXON_HTTP_RPC_URL: https://rpc-alphanet-axon.ckbapp.dev
```

Fix AXON_HTTP_RPC_URL